### PR TITLE
[NUI] Fix build warning for AccessibilityEvent naming

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
- There's a build warning for View.Activate EventHandler : CS0108.
 'GaussianBlurView.Activate()' hides inherited member 'View.Activate'.
  Use the new keyword, 'Activated'.
- Add 'Accessibility' prefix to all Accessibility events.
- Fix build warning : CA2000
  Call Dispose on object created by signals before all references are out of scope.
  -> Make signal variable separately.
- Update some code to follow formatting rules.

### API Changes ###
- N/A
-

### CODE REVIEW

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Review by hyunjushin — APPROVED** [review]

**Comment by huiyueun** [comment]
I think it's better to add 'Accessibility' to event name. 'AccessibilityActivated'

File: src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs [hunk 1]
```diff
@@ -391,33 +421,39 @@ internal VoidSignal ReadingSkippedSignal()
 
         private VoidHandlerType readingPausedCallback;
         private EventHandler readingPausedHandler;
+        internal VoidSignal readingPausedSignal;
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs:424]: internal의 경우 대문자로 변경해주세요.
https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=191373542

Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs]: 넵 모두 바꿔서 push 하였습니다!!!

**Review by huiyueun — APPROVED** [review]

**Pull request merged**